cirq-google/cirq_google/calibration/engine_simulator.py

```diff
@@ -304,6 +305,45 @@ def create_from_characterizations_sqrt_iswap(
             ideal_when_missing_parameter=ideal_when_missing_parameter,
         )
 
+    @classmethod
+    def create_from_gate_dict(
+        cls,
+        gates_dict: Dict[
```

Comment:
It would be good to keep the argument names consistent, with this one just being `parameters`. There are also other arguments in the original method like `simulator` and `ignore_`, can you add them here as well? This is externally visible method.

In a follow up we can remove all the logic from `create_from_dictionary_sqrt_iswap` and just delegate to this method but in order to do so, this needs to be a generalization of the other one.

Reply:
Renamed gates_dict->parameters and added simulator.
What do you mean by `ignore_`. Are you referring to `ideal_when_missing_gate` and `ideal_when_missing_parameter`? Do we really need them here? 

As far as i'm concerned, I'm adding this factory method just so I can write a test, so I am passing only these parameters which I need. In my test I want to require that all FSim-like gates in circuit must be explicitly specified in engine simulator.